webbpsf/trending.py

```diff
@@ -941,30 +1350,43 @@ def get_month_start_end(year, month):
     return start_date, end_date
 
 
```

Comment:
filter_opdtable_for_month is new method to be used for auto generating, and filtering existing opdtable